frontend/src/components/common/styles/paginator.scss

```diff
@@ -1,5 +1,6 @@
-@import 'include';
+@use 'sass:math';
```

Comment:
This fixes a deprecation warning, the division syntax is going away in future versions of sass